dev/puppet-server.conf.sample

```diff
@@ -30,8 +30,11 @@ webserver: {
 # configure the mount points for the web apps
 web-router-service: {
     "puppetlabs.services.ca.certificate-authority-service/certificate-authority-service": "/puppet-ca"
-    "puppetlabs.services.master.master-service/master-service": "/puppet"
+    "puppetlabs.services.master.master-service/master-service": {master-routes:       "/puppet"
 
+                                                                 # The following controls 404 errors and should not
+                                                                 # be modified
+                                                                 invalid-in-puppet-4: "/" }
```

Comment:
Similar to comments Ruth made in the MRI pull request, will this route catch all invalid requests, or only the subset of invalid requests that happen to be previously valid?  If it's going to catch all invalid requests, it should be changed to only display the "upgrade" message when the request is from an agent that previously was valid.